angular.json

```diff
@@ -93,7 +93,11 @@
                 "output": "./svg"
               }
             ],
-            "styles": ["apps/cookbook/src/styles.scss", "node_modules/prismjs/themes/prism.css"],
+            "styles": [
+              "libs/designsystem/src/lib/fonts/roboto.css",
+              "apps/cookbook/src/styles.scss",
```

Comment:
The fonts should be imported through `styles.scss`, alternatively through `global-styles.scss`

Reply:
The Cookbook now imports the fonts through `styles.scss` by `_global-styles.scss` which in turn imports `.../base/_fonts.scss`, and readme is updated 😄